schema_enforcer/schemas/jsonschema.py

```diff
@@ -32,6 +32,7 @@ def __init__(self, schema, filename, root):
         ]
         self.validator = None
         self.strict_validator = None
+        self.format_checker = draft7_format_checker
```

Comment:
please can you move it outside of init, at the root of the class, since its value is always the same.

Reply:
I don't think this is a good idea; if someone wanted to monkey patch and use a different draft, then that would change it for every instance.

Reply:
I'm not gonna argue this one but if someone need to monkey patch this class we should revisit the design altogether.